Update "Our Team" page

hanboyu opened the pull request:
# Summary
Closes #1353 

Updated "Our Team" page according to the Figma design:  https://www.figma.com/file/vH0gBrC6f6VctocGNhhWSD/Teams-Page?type=design&node-id=301-2112&mode=design&t=Tt9T6EDbk437f5v3-0 

# Checklist

- [ ] On the frontend, I've made my strings translate-able.
- [ ] If I've added shared components, I've added a storybook story.
- [X] I've made pages responsive and look good on mobile.

# Screenshots
## Desktop View:
![screencapture-localhost-3000-about-our-team-2023-10-26-15_32_12](https://github.com/codeforboston/maple/assets/39149005/4526e01f-ae70-428e-b2e6-2866b9577d48)
![screencapture-localhost-3000-about-our-team-2023-10-26-15_32_47](https://github.com/codeforboston/maple/assets/39149005/9c8f701c-eff1-4323-b64e-1ab1e68fc159)
![screencapture-localhost-3000-about-our-team-2023-10-26-15_33_01](https://github.com/codeforboston/maple/assets/39149005/2c128e4f-de39-4a22-8881-d3af69a4d69b)
## Mobile View:
![screencapture-localhost-3000-about-our-team-2023-10-26-15_33_31](https://github.com/codeforboston/maple/assets/39149005/4567f7bf-1de7-45b8-bbdd-b628cdb996b6)
![screencapture-localhost-3000-about-our-team-2023-10-26-15_33_59](https://github.com/codeforboston/maple/assets/39149005/3bf5c857-675d-4509-bb00-d2a876680491)
![screencapture-localhost-3000-about-our-team-2023-10-26-15_34_28](https://github.com/codeforboston/maple/assets/39149005/3e9a54b9-969b-40cd-9544-319fd8c7de4c)
![image](https://github.com/codeforboston/maple/assets/39149005/3389c28c-1b8a-4b6f-af3e-14f05cebfa3e)


# Steps to test/reproduce
1. Go to the home page
2. Click "About" -> "Our Team" in the menu

mvictor55 commented:
@hanboyu looks great!! One comment -- we forgot to change the description for the Advisory Board. Can you update it to reflect this: 
<img width="339" alt="image" src="https://github.com/codeforboston/maple/assets/71357256/c4e4df01-08cd-4734-ba9a-bff7905bee41">


I've also updated the figma desktop version

nesanders commented:
Looking good!  Thank you @hanboyu 

A few questions, partly for @mvictor55 

* Do we want to hyperlink (`mailto`) the email addresses?
* Do we want to hyperlink to any of our team/advisor's homepages or to the websites of their affiliated orgs? Not sure if the original bios we collected had links or not.

mvictor55 commented:
I think we should try to hyperlink our own emails. But, for now, don’t need
to adjust the links in our advisors’ profiles

On Thu, Oct 26, 2023 at 4:07 PM Nathan E. Sanders ***@***.***>
wrote:

> Looking good! Thank you @hanboyu <https://github.com/hanboyu>
>
> A few questions, partly for @mvictor55 <https://github.com/mvictor55>
>
>    - Do we want to hyperlink (mailto) the email addresses?
>    - Do we want to hyperlink to any of our team/advisor's homepages or to
>    the websites of their affiliated orgs? Not sure if the original bios we
>    collected had links or not.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/codeforboston/maple/pull/1362#issuecomment-1781823919>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARANGSCHSHQ75SYXI7TLEV3YBK7HFAVCNFSM6AAAAAA6RXPARCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBRHAZDGOJRHE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

hanboyu commented:
> @hanboyu looks great!! One comment -- we forgot to change the description for the Advisory Board. Can you update it to reflect this: <img alt="image" width="339" src="https://user-images.githubusercontent.com/71357256/278461614-c4e4df01-08cd-4734-ba9a-bff7905bee41.png">
> 
> I've also updated the figma desktop version

Opps, I just fixed it.

hanboyu commented:
> I think we should try to hyperlink our own emails. But, for now, don’t need to adjust the links in our advisors’ profiles
> […](#)
> On Thu, Oct 26, 2023 at 4:07 PM Nathan E. Sanders ***@***.***> wrote: Looking good! Thank you @hanboyu <https://github.com/hanboyu> A few questions, partly for @mvictor55 <https://github.com/mvictor55> - Do we want to hyperlink (mailto) the email addresses? - Do we want to hyperlink to any of our team/advisor's homepages or to the websites of their affiliated orgs? Not sure if the original bios we collected had links or not. — Reply to this email directly, view it on GitHub <[#1362 (comment)](https://github.com/codeforboston/maple/pull/1362#issuecomment-1781823919)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ARANGSCHSHQ75SYXI7TLEV3YBK7HFAVCNFSM6AAAAAA6RXPARCVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTOOBRHAZDGOJRHE> . You are receiving this because you were mentioned.Message ID: ***@***.***>

Currently, the emails in "Steering Committee" are not hyperlinked, I can quickly change that.

mertbagt commented:
:rocket:
